dbfit-java/build.gradle

```diff
@@ -222,3 +251,106 @@ project('netezza') {
     }
 }
 
+// Aggregated dependencies to include in the POM file when publishing.
+dependencies {
+    compile publicCoreCompileDeps
+    runtime publicHsqldbRuntimeDeps
+    compile publicOracleCompileDeps
+    compile publicMysqlCompileDeps
+    compile publicPostgresCompileDeps
+}
+
+// Remove the default artifact as there are no Java sources for this project.
+configurations.archives.artifacts.clear()
+
+task allJavadoc(type: Javadoc) {
+    source subprojects.collect {
+        it.sourceSets.main.allJava
+    }
+    destinationDir = new File(buildDir, 'doc/javadoc')
+    classpath = files(subprojects.collect {
+                          it.sourceSets.main.compileClasspath
+                      }
+                     )
+}
+
+task allJavadocJar(type: Jar) {
+    baseName = 'dbfit'
+    classifier = 'javadoc'
+    from allJavadoc
+}
+
+task allSourcesJar(type: Jar) {
+    baseName = 'dbfit'
+    classifier = 'sources'
+    subprojects.each {
+        from it.sourceSets.main.allSource
+    }
+}
+
+task allJar(type: Jar, dependsOn: subprojects.build) {
+    baseName = 'dbfit'
+    subprojects.each { subproject ->
```

Comment:
Well, it does seem to work (see below) but is it (use of `it` in outer and inner iterations) too confusing?

```groovy
task allJar(type: Jar, dependsOn: subprojects.build) {
    baseName = 'dbfit'
    subprojects.each {
        from it.configurations.archives.allArtifacts.files.collect {
            zipTree(it)
        }
    }
}